dashboard/test/models/user_helpers_test.rb

```diff
@@ -83,9 +89,9 @@ def create_user_with_username(username)
   end
 
   test 'generate_username for parentheses and apostrophes' do
-    assert_equal 'kermit_the_frog',
-      UserHelpers.generate_username(User, 'Kermit (the frog)')
```

Comment:
Why do we simplify this input?

Reply:
Because of the whitespaces in the original. `Kermit (the frog)` will become the username `kermit` and therefore not test the handling of parenthesis in a name.